specs/pvs/corla.pvs

```diff
@@ -1193,10 +1205,18 @@ BEGIN
 %%% may establish different risk limits for comparison audits and
 %%% ballot polling audits, but in no event will the risk limit exceed
 %%% five percent.
-  establish_risk_limit_comparison_audits:
-    [department_of_state_dashboard, {r: risk_limit | r <= 5} -> department_of_state_dashboard]
-  establish_risk_limit_ballot_polling_audits:
-    [department_of_state_dashboard, {r: risk_limit | r <= 5} -> department_of_state_dashboard]
+  establish_risk_limit_for_comparison_audits:
+    [department_of_state_dashboard, {r: risk_limit | r <= 5} ->
+     department_of_state_dashboard]
+  establish_risk_limit_for_ballot_polling_audits:
+    [department_of_state_dashboard, {r: risk_limit | r <= 5} ->
```

Comment:
Since CDOS retains the right to change anything in the real audit, I'd leave out an enforced check on the risk limit being less than 5%. Both counties and CORLA2 have supported a move to 10% for county-level contests, and they may want flexibility.